src/Application/test/AddCalculatorGTest.cpp

```diff
@@ -0,0 +1,45 @@
+///////////////////////////////////////////////////
+/**
+ * @file AddCalculatorGTest.cpp
+ * @brief Unit tests for Add Calculator
+ */
+///////////////////////////////////////////////////
+
+#include<gtest/gtest.h>
+#include<gmock/gmock.h>
+#include<limits>
+
+#include "AddCalculator.h"
+#include "CalculatorApplication.h"
+#include "CalculatorApplicationFactory.h"
+#include "CalculatorStrings.h"
+
+using namespace calculator;
+using namespace testing;
+
+class whenTestingAddCalculator : public :: testing :: Test
```

Comment:
If you aren't using the fixture... just remove this class and then change all of your `TEST_F(...)` to `TEST(...)`

Reply:
You still should remove this class. I.e., just delete lines 20-23

And do the same with all the other tests likes this.